### PR TITLE
Feature #125/occupied charging points retail

### LINKED ISSUE #125: change logic for occupied charging points in use-case retail

### PR DESCRIPTION
**Pull request by JakobWeg**
Closes #125

### CODE REVIEW

**Review by mosc5 — COMMENTED** [review]
Have you tested if this still leads to consistent event outputs? Park event start plus event time should be equal to the start of the next event. If not we could add a parking event after the charging

File: simbev/car.py [hunk 1]
```diff
@@ -933,18 +933,6 @@ def export(self, region_directory, simbev):
...
-                    charge_event["park_ts_end"]
-                    if (charge_event["park_ts_end"] - charge_event["start"])
-                    * simbev.step_size
-                    / 60
-                    <= simbev.occupation_time_max
-                    else int(
-                        charge_event["start"]
-                        + simbev.occupation_time_max * 60 / simbev.step_size
-                    )
-                )
```
Comment (mosc5) [inline simbev/car.py:949]: What did this do and why is it removed?

File: simbev/region.py [hunk 2]
```diff
@@ -224,12 +224,12 @@ def update_grid_timeseries(
...
-                                                                      (24 * 60 / self.region_type.step_size))
-                        frac_park_end, whole_park_end = math.modf(park_ts_end / (24 * 60 / self.region_type.step_size))
-                        if whole_park_start < whole_park_end:
-                            park_ts_end = int((whole_park_start + 1) * (24 * 60 / self.region_type.step_size))
+                    #if use_case == "retail":
+                    #    frac_park_start, whole_park_start = math.modf(timestep_start /
+                    #                                                  (24 * 60 / self.region_type.step_size))
+                    #    frac_park_end, whole_park_end = math.modf(park_ts_end / (24 * 60 / self.region_type.step_size))
+                    #    if whole_park_start < whole_park_end:
+                    #        park_ts_end = int((whole_park_start + 1) * (24 * 60 / self.region_type.step_size))
```
Comment (mosc5) [inline simbev/region.py]: Remove if not needed. This section might also benefit from comments

File: simbev/region.py [hunk 3]
```diff
@@ -223,6 +224,12 @@ def update_grid_timeseries(
             if code in self.header_grid_ts:
                 column = self.header_grid_ts.index(code)
                 if i == 0:
+                    #if use_case == "retail":
+                    #    frac_park_start, whole_park_start = math.modf(timestep_start /
+                    #                                                  (24 * 60 / self.region_type.step_size))
+                    #    frac_park_end, whole_park_end = math.modf(park_ts_end / (24 * 60 / self.region_type.step_size))
+                    #    if whole_park_start < whole_park_end:
+                    #        park_ts_end = int((whole_park_start + 1) * (24 * 60 / self.region_type.step_size))
```
Comment (mosc5) [inline simbev/region.py]: ?

File: simbev/car.py (hunk 1 above)
Comment (JakobWeg) [reply simbev/car.py:949]: This limits the maximum occupation_time of a charging-event. Now the charging_event it self is limited, so this is not needed anymore.

**Pull request merged**